_layouts/frontpage.html

```diff
@@ -75,7 +75,6 @@ <h1>{{page.headerTitle}}</h1>
               <img src="/resources/img/frontpage/scala-spiral.png" alt="">
               <div class="scala-version">
                 <span>Scala</span>
-                <span>{{page.currentScalaVersion}}</span>
```

Comment:
Why removing the current scala version?

Reply:
Which one should we show? Scala 2 or Scala 3, maybe both? I just deleted it since the buttons now already show the versions.

Reply:
OK, I didn’t see that. If it’s easy to show both Scala 2 and Scala 3 version I would go for that, otherwise I think we should show the Scala 3 version number.

Reply:
Just to be clear: you think showing them in the buttons only is not enough? 

![image](https://user-images.githubusercontent.com/408265/118698723-0901cd80-b811-11eb-9971-1b83458cd176.png)

If no then I can also add the versions here tomorrow.

Reply:
No, OK it’s enough to have the version on the buttons.